user_guide/intro_nav_sidebar.rst

```diff
@@ -0,0 +1,254 @@
+.. _user-guide-navigation-sidebar-panel:
+  
+Sidebar Panel
+=============
+
+The sidebar panel is a host for a number of widgets that provide fast and convenient access to frequently needed 
+information, such as recent emails or the task list. 
+
+The sidebar panel is by default located on the right side of the 
+page, however, subject to configuration settings of your organization, there can be a sidebar on the left, two sidebars 
+(on the left and on the right) and no sidebar at all. Location of the sidebar is defined by the system administrator for 
+the whole organization. 
```

Comment:
Edit: By default, the sidebar panel is located on the right side of the page. However, subject to the configuration settings of your organization, there can be a sidebar on the left, two sidebars (on the left and on the right), or no sidebar at all. The location of the sidebar is defined by the system administrator for the whole organization.